addons/website/views/snippets/s_process_steps.xml

```diff
@@ -1,15 +1,34 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+<template id="s_process_steps_options" inherit_id="website.snippet_options">
+    <xpath expr="." position="inside">
+        <div data-js="ConnectorChoice" data-selector=".s_process_steps">
+            <we-row string="Connector">
+                <we-colorpicker 
+                    data-select-style="true"
+                    data-css-property="border-top-color"
+                    data-color-prefix="border-"
+                    data-apply-to=".s_process_step_line_left, .s_process_step_line_right"/>
+                <we-select>
+                    <we-button data-change-connector="none">None</we-button>
+                    <we-button data-change-connector="line">Line</we-button>
+                    <we-button data-change-connector="arrow">Straight arrow</we-button>
+                    <we-button data-change-connector="curved_arrow">Curved arrow</we-button>
+                </we-select>
```

Comment:
I would prefer the select, then the colorpicker. Don't you agree?
And also, the colorpicker should not appear when the "none" connector is chosen.